impeller/display_list/canvas.cc

```diff
@@ -1115,24 +1116,37 @@ void Canvas::SaveLayer(const Paint& paint,
       input_texture = backdrop_data->texture_slot;
     }
 
-    backdrop_filter_contents = backdrop_filter_proc(
-        FilterInput::Make(std::move(input_texture)),
-        transform_stack_.back().transform.Basis(),
-        // When the subpass has a translation that means the math with
-        // the snapshot has to be different.
-        transform_stack_.back().transform.HasTranslation()
-            ? Entity::RenderingMode::kSubpassPrependSnapshotTransform
-            : Entity::RenderingMode::kSubpassAppendSnapshotTransform);
-
     if (will_cache_backdrop_texture) {
       FML_DCHECK(backdrop_data);
       // If all filters on the shared backdrop layer are equal, process the
       // layer once.
       if (backdrop_data->all_filters_equal &&
           !backdrop_data->shared_filter_snapshot.has_value()) {
-        // TODO(157110): compute minimum input hint.
+        Entity snapshot_entity;
```

Comment:
I _think_ the coverage hint parameter of RenderToSnapshot accepts a source space rectangle. In which case, i think you can *just* uninion all the coverage rects and adjust the offset at the end.